packages/styles/src/icons/_settingsBusiness.scss

```diff
@@ -236,5 +237,7 @@ $fd-business-suite-icons: (
   cross-union: "\e182",
   empty-union: "\e183",
   cross-target: "\e184",
-  non-stackable: "\e185"
+  non-stackable: "\e185",
+  back-to-search: "\e186",
+  business-objects-folder: "\e187"
```

Comment:
please add a comma at the end, so next update doesn't detect this as a new change